.github/workflows/action.yml

```
@@ -0,0 +1,26 @@
+name: Action
```

```suggestion
name: Test Action
```
(or something along those lines)